### ISSUE TITLE
RegExp literals

### ISSUE THREAD

**Issue by GoogleCodeExporter** [neades/yaji-ecmascript-interpreter#9]
```
Specification: ES5
Reference: 7.8.5

Description
RegExp literals to be implemented

RegExp should no longer be an optional package
```

Original issue reported on code.google.com by `neil.ead...@gmail.com` on 7 Jun 2011 at 10:26
- Blocked on: #156

**Comment by GoogleCodeExporter** [comment]
```
Issue 42 has been merged into this issue.
```

Original comment by `neil.ead...@gmail.com` on 19 Jun 2011 at 9:50

**Comment by GoogleCodeExporter** [comment]
Original comment by `neil.ead...@gmail.com` on 25 Aug 2011 at 12:33

**Comment by GoogleCodeExporter** [comment]
```
Status update
```

Original comment by `neil.ead...@gmail.com` on 30 Aug 2011 at 1:48
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
The change for this is done now. r88
```

Original comment by `neil.ead...@gmail.com` on 31 Aug 2011 at 9:39
- Changed state: **Fixed**
